docs/modules/ROOT/pages/how-tos/configuring/entitlement_subscription.adoc

```diff
@@ -0,0 +1,28 @@
+= Using Red Hat entitlement subscription
```

Comment:
Is this something that we want to include in the upstream documentation? It feels very specific to our own use cases.

Reply:
Can someone else give some feedback?

Reply:
Aren't subscriptions in fact more aimed at external customers? Red Hatters usually have ways to get around needing subscriptions

Reply:
But maybe we could put it in some "If you're building with Red Hat software" section to make it less in-your-face

Reply:
I would keep the changes in this pr unchanged, since it is just for moving the documents. For any other decision and related changes, we need new stories.